sources/packages/web/src/views/Student.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <Card class="p-m-4">
+    <template #title>
+      Student Information
+    </template>
+    <template #content>
+    <!--The information from Services Card includes First Name, Last Name, Middle Name, Date of Birth, 
+     verified email, and Gender as read-only-->
+      
+    <div class="p-fluid p-formgrid p-grid">
+      <div class="p-field p-col">
+          <label for="firstName">First Name</label>
+          <InputText id="firstName" v-model="studentProfile.displayName" readonly />
+      </div>
+      <div class="p-field p-col">
+          <label for="familyName">Family Name</label>
```

Comment:
Consider change "Family Name" to "Last Name" as described in the story.

Reply:
Done